pipelines/skyPlotExample.yaml

```diff
@@ -81,3 +81,34 @@ tasks:
       python: |
         from lsst.analysis.drp.dataSelectors import PsfFlagSelector, HighSnSelector, GalaxyIdentifier
         from lsst.analysis.drp.calcFunctors import SNCalculator
+
+ # Same as above but this time plot it for galaxies not stars
+ # The defualt is stars but you can also use all for both.
+ # Then adjust the SN threshold for points to be included in the plot.
+  plot_psfSn_goodCmodel_stars_scatter:
+    class: lsst.analysis.drp.scatterPlot.ScatterPlotWithTwoHistsTask
+    config:
+      axisActions.yAction: SNCalculator
```

Comment:
I feel like somewhere in this example (perhaps in the comment preceding this block) it should explain that there is a default `xAction` and what it is, and that's why no `xAction` is provided on this scatterplot example.